### PR TITLE
Elastic cluster doc 1012

### PR DESCRIPTION
**Pull request by lennessyy**
Description document for ER cluster

### CODE REVIEW

File: modules/ha/pages/elastic-cluster.adoc [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+The ER cluster shares storage with the primary cluster.
+When the ER cluster is set up, it reads from the shared storage to retrieve all data from the primary cluster.
+After setup, it no longer receives updates from the primary cluster.
+
+.Architectural diagram for ER cluster
+image:elastic-cluster.png[]
+
+The ER clusters are able to process memory-intensive OLAP queries in an isolated environment with no impact to the primary cluster.
+After the queries finish, the ER cluster should be shut down.
+The next time when there is a demand for a memory-intensive OLAP query, you can set up a new ER cluster to meet the demand.
```
Comment (jinyan1997) [inline modules/ha/pages/elastic-cluster.adoc:30]: Should we mention the setup process? such as involve support and etc?

I think the point is to mention that the setup is going to be (RTM release) a few button clicking and light wegihted.

File: modules/ha/pages/elastic-cluster.adoc [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+== Use cases
+If your workload involves online analytical processing (OLAP) queries that access large amounts of data and are very time-consuming, you can benefit from using ER clusters for your OLAP queries.
```
Comment (jinyan1997) [inline modules/ha/pages/elastic-cluster.adoc:33]: Another huge impact scenario is to allow running OLAP queries while primary cluster is taking a backup.

File: modules/release-notes/pages/index.adoc [hunk 3]
```diff
@@ -12,6 +12,11 @@ Below is a list of new features and improvements in TigerGraph 3.5.0:
 === Ecosystem integration
 * Introduced the xref:data-loading:data-streaming-connector.adoc[Data Streaming Connector] with Google Cloud Storage data streaming support.
 
+=== Elasticity
+* Added support for provisioning xref:ha:elastic-cluster.adoc[Elastic Read-only (ER) clusters] to provide computation on demand.
+** Note: This feature is currently in the preview stage.
+To set up an ER cluster for your environment, contact mailto:support@tigergraph.com[].
```
Comment (jinyan1997) [inline modules/release-notes/pages/index.adoc:18]: Currently only supported on Paid instance.

File: modules/ha/pages/elastic-cluster.adoc (hunk 1 above)
Comment (lennessyy) [reply modules/ha/pages/elastic-cluster.adoc:30]: I think at this stage, just telling people to contact support for setup is sufficient. I have a note in the beginning to tell people to contact support if they want to use the feature. When the final version is ready, we'll change this.

**Review by jinyantg2103 — APPROVED** [review]

File: modules/ha/pages/elastic-cluster.adoc [hunk 4]
```diff
@@ -0,0 +1,59 @@
+= Compute-on-demand Elastic Cluster
```
Comment (songting-chen) [inline modules/ha/pages/elastic-cluster.adoc:1]: can't comment on the figure.

My suggestion: 
Primary Production Cluster

File: modules/ha/pages/elastic-cluster.adoc [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+:description: Overview of TigerGraph's compute-on-demand ER cluster.
```
Comment (songting-chen) [inline modules/ha/pages/elastic-cluster.adoc:2]: Soon we will have clone cluster - maybe we should ask PM for naming

**Review by songting-chen — APPROVED** [review]

File: modules/ha/pages/elastic-cluster.adoc [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+TigerGraph allows you to spin up an Elastic Read-only (ER) cluster from your primary cluster to handle compute-intensive online analytical processing (OLAP) queries on demand.
+Operations on the ER cluster do not interfere with the primary cluster in any way.
+
+NOTE: This feature is currently in alpha.
```
Comment (arunramasami) [inline modules/ha/pages/elastic-cluster.adoc]: Let us not use the alpha term here. Instead, use "preview version" for the early adopters/customers

Comment (lennessyy) [reply modules/ha/pages/elastic-cluster.adoc]: Yes, I think I had already changed to "preview version" as I realized that alpha is an inappropriate term

**Pull request merged**